app/src/main/res/xml/xdrip_plus_prefs.xml

```diff
@@ -424,6 +424,14 @@
                     android:key="illustrate_remote_data"
                     android:summary="Show data received remotely (diagnostic only)"
                     android:title="Illustrate remote data" />
+                <CheckBoxPreference
```

Comment:
Can I suggest instead of having these as checkbox preferences, having a numeric value for these settings fields that defaults to 40 for the low and 250 for the high? (the current defaults)

As-is these are sort of confusing, since it's a checkbox for changing a number between two presets.

Reply:
The defaults are 40 and 250.  

I can change it to what you suggest still keeping the defaults at 40 and 250.  
But, I'm not sure someone else is not going to prefer with checkboxes.

Reply:
I changed the settings to lists.  There are now multiple options for each with 40 and 250 as defaults.